nevergrad/benchmark/experiments.py

```diff
@@ -326,6 +333,14 @@ def yabbob(seed: tp.Optional[int] = None, parallel: bool = False, big: bool = Fa
                     yield xp
 
 
+@registry.register
+def yadimbudgetbbob(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
+    """Yabbob variant aimed at exploring the link between dimension and budget specifically."""
+    internal_generator = yabbob(seed, parallel=False, big=True, hd=True)
```

Comment:
I don't think this case is interesting: this is not parallel, so Powell will win whatever happens as far as I understing.
If you turn it on, the num_workers will be 100 and it wont make sense either for budgets smaller than 500